### ISSUE TITLE
TestBootstrapPolicyOverwritePolicyCommand can flake on slow cache

### ISSUE THREAD

**Issue by deads2k** [openshift/origin#2956]
If the policy cache doesn't see the policy deletion before the list runs, you can get a success.

```
[1.052332ms] [1.050232ms] Deep copied
[1.055944ms] [3.612µs] END
E0608 18:54:00.434573   21653 reflector.go:133] Failed to list *api.Pod: User "system:openshift-client" cannot list all pods in the cluster
I0608 18:54:01.237626   21653 endpoints_controller.go:258] Finished syncing service "default/kubernetes-ro" endpoints. (14.936µs)
I0608 18:54:01.237619   21653 endpoints_controller.go:258] Finished syncing service "default/kubernetes" endpoints. (26.314µs)
--- FAIL: TestBootstrapPolicyOverwritePolicyCommand-32 (12.32s)
	bootstrap_policy_test.go:95: unexpected error: <nil>
FAIL

```

**Comment by smarterclayton** [comment]
Fixinf

**Comment by smarterclayton** [comment]
Fixed in #3033

**Comment by deads2k** [comment]
@liggitt please close
